docs/maui/views/popup.md

```diff
@@ -0,0 +1,206 @@
+
+```csharp
+void OnYesButtonClicked(object? sender, EventArgs e) => Close(true);
+
+void OnNoButtonClicked(object? sender, EventArgs e) => Close(false);
+```
+
+The `Close` method allows for an `object` value to be supplied, this will be the resulting return value. In order to await the result the `ShowPopupAsync` method must be used as follows:
+
+```csharp
+using CommunityToolkit.Maui.Views;
+
+public class MyPage : ContentPage
+{
+    public async Task DisplayPopup()
+    {
+        var popup = new SimplePopup();
+
+        var result = await ShowPopupAsync(popup);
+
+        if (result is bool boolResult)
+        {
+            if (boolResult)
+            {
+                // Yes was tapped
+            }
+            else
+            {
+                // No was tapped
+            }
+        }
+    }
+}
+```
+
+> [!NOTE]
+> In order to handle the tapping outside of a `Popup` when also awaiting the result you can change the value that is returned through the `ResultWhenUserTapsOutsideOfPopup` property.
+
+## Properties
+
+|Property  |Type  |Description  |
+|---------|---------|---------|
+| `Anchor` | `View` | Gets or sets the `View` anchor. The Anchor is where the Popup will render closest to. When an Anchor is configured the popup will appear centered over that control or as close as possible. |
+| `CanBeDismissedByTappingOutsideOfPopup` | `bool` | Gets or sets a value indicating whether the popup can be dismissed by tapping outside of the Popup. On Android - when false the hardware back button is disabled. |
+| `Color` | `Color` | Gets or sets the `Color` of the Popup. This color sets the native background color of the `Popup`, which is independent of any background color configured in the actual `Content`. |
+| `Content` | `View` | Gets or sets the `View` content to render in the `Popup`. |
+| `HorizontalOptions` | `LayoutAlignment` | Gets or sets the `LayoutAlignment` for positioning the `Popup` horizontally on the screen. |
+| `Result` | `Task<object?>` | Gets the final result of the dismissed `Popup`. |
+| `Size` | `Size` | Gets or sets the `Size` of the Popup Display. The Popup will always try to constrain the actual size of the `Popup` to the size of the View unless a `Size` is specified. If the `Popup` uses the `HorizontalOptions` or `VerticalOptions` properties that are not the defaults then this `Size` property is required. |
+| `VerticalOptions` | `LayoutAlignment` | Gets or sets the `LayoutAlignment` for positioning the `Popup` vertically on the screen. |
+
+## Events
+
+|Event | Description  |
+|---------|---------|
+| `Closed` | The event that is dismissed event is invoked when the `Popup` is closed. |
+| `Opened` | The event that is dismissed event is invoked when the `Popup` is opened. |
+
+## Examples
+
+You can find an example of this converter in action in the [.NET MAUI Community Toolkit Sample Application](https://github.com/CommunityToolkit/Maui/blob/main/samples/CommunityToolkit.Maui.Sample/Pages/Views/).
```

Comment:
```suggestion
You can find an example of this converter in feature in the [.NET MAUI Community Toolkit Sample Application](https://github.com/CommunityToolkit/Maui/blob/main/samples/CommunityToolkit.Maui.Sample/Pages/Views/).
```

I love how you updated it for toast and snackbar above and then miss this one yourself 😝

Reply:
Doh! Thanks for the catch

Reply:
![image](https://user-images.githubusercontent.com/17139988/163553978-075d811c-2156-4d7a-a486-ca667edace54.png)

It looks like we both had a slight mess up here 😂

Reply:
NOOOOOOO! Haha 🤦